Automattic/mydb-observer
# Remove `101` package and replace functions used from it.

`101` was being used for its `omit` and `put` functions which I've replaced with modern js equivalents.

* `omit` will remove a key/value pair from an object based on its key name.
* `put` clones an object and adds a new key/value pair to it.

**To Test**
* `rm -rf node_modules`
* `npm i`
* Start `cloudup-vm` to fire up the db server
* Run `make test`, the tests should pass (I had to `control-c` stop the test run, any ideas on how to fix?)
* `npm audit` should be clean 😍
* Use this branch in `cloudup-vm`. You should see the `mydb-observer emitted` events firing normally and Cloudup should work 😉
